README.md

```diff
@@ -63,3 +73,29 @@ Each individual plugin can be disabled, replaced or used with other plugins in t
   }
 }
 ```
+
+The `npmPublish` and `tarballDir` option can be used to skip the publishing to the `npm` registry and instead, release the package tarball with the [github](https://github.com/semantic-release/github) and [git](https://github.com/semantic-release/git) plugins:
+
+```json
+{
+  "release": {
+    "verifyConditions": ["@semantic-release/conditions-travis", "@semantic-release/npm", "@semantic-release/git", "@semantic-release/github"],
+    "getLastRelease": "@semantic-release/npm",
+    "publish": [
+      {
+        "path": "@semantic-release/npm",
+        "npmPublish": false,
+        "tarballDir": "dist"
+      },
+      {
+        "path": "@semantic-release/git",
+        "assets": "dist/*.tgz"
+      },
+      {
+        "path": "@semantic-release/github",
+        "assets": "dist/*.tgz"
+      },
+    ]
+  }
+}
```

Comment:
This will commit the tarbal to git as well as upload it to the  release on GitHub, right? I’m not sure if that makes a lot of sense? Wouldn’t just the upload to the github release be the more common use case?

Reply:
> This will commit the tarbal to git as well as upload it to the release on GitHub, right?

Yes

> I’m not sure if that makes a lot of sense? Wouldn’t just the upload to the github release be the more common use case?

The intention was to demonstrate the capability more than recommending a specific configuration. But that might be confusing indeed.
I'll change it to upload only to only upload to github.

Reply:
Done